### PR TITLE
Readd PDF services

### LINKED ISSUE #31: Readd PDF Services

### PR DESCRIPTION
**Pull request by TheFehr**
Closes #31

### CODE REVIEW

File: app/models/organization_member.rb [hunk 1]
```diff
@@ -3,6 +3,14 @@
 class OrganizationMember < ApplicationRecord
   belongs_to :organization
   has_one :user, as: :referencee, dependent: :destroy, required: false, autosave: true
+  has_one :service_specification_contact_person,
```
Comment (TheFehr) [inline app/models/organization_member.rb:6]: @lukasbischof shouldn't thid actually be has_many? One OrganizationMember can be added to more then one service specification.

Comment (lukasbischof) [reply app/models/organization_member.rb:6]: How would this then be represented on the print version?

Comment (TheFehr) [reply app/models/organization_member.rb:6]: The specification still has one lead and one contact person. But this right now limits them to be added to one specification max. Or am I totally confused?

File: app/services/pdfs/expense_sheet/fields/info_block.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        FOOTER_ROWS = [
+          {
+            label: "<b>#{I18n.t('activerecord.attributes.civil_servant.iban')}:</b>",
+            content: lambda do |expense_sheet|
+              "<b>#{IBANTools::IBAN.new(expense_sheet.service.civil_servant.iban).prettify}</b>"
+            end
+          },
+          {
+            label: "<b>#{I18n.t('pdfs.expense_sheet.info_block.footer.bank_account_number')}:</b>",
+            content: '4470 (200)'
```
Comment (lukasbischof) [inline app/services/pdfs/expense_sheet/fields/info_block.rb:62]: Do we even need that?

File: spec/services/pdfs/expense_sheet/generator_service_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = 1000
```
Comment (lukasbischof) [inline spec/services/pdfs/expense_sheet/generator_service_spec.rb:3]: Wat dis?

File: spec/services/pdfs/expense_sheet/generator_service_spec.rb [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
```
Comment (lukasbischof) [inline spec/services/pdfs/expense_sheet/generator_service_spec.rb:4]:
```suggestion
```

File: spec/services/pdfs/phone_list_service_spec.rb [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+RSpec.describe Pdfs::PhoneListService, type: :service do
+  describe '#render' do
+    context 'when locale is german' do
+      before do
+        I18n.locale = :de
+        create(:service, service_data)
+      end
+
+      after { I18n.locale = I18n.default_locale }
```
Comment (lukasbischof) [inline spec/services/pdfs/phone_list_service_spec.rb:13]:
```suggestion
      around { |spec| I18n.with_locale(:de) { spec.run } }

      before { create(:service, service_data) }
```

File: app/services/pdfs/expense_sheet/fields/info_block.rb (hunk 2 above)
Comment (TheFehr) [reply app/services/pdfs/expense_sheet/fields/info_block.rb:62]: We will need to check this later #65

**Pull request merged**